### PR TITLE
Added an example of using Clojure Unmanaged Extension

### PR DESCRIPTION
**Pull request by ducky427**

### CODE REVIEW

File: language-guides/clojure/server-extension/src/example/core.clj [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+(deftype ^{Path "/"} MovieResource []
+         IMovieResource
+         (^{GET true
+            Produces ["text/plain"]
+            Path "/search"}
+          search
+          [this ^{Context true} database ^{QueryParam "q"} query]
+            (require 'example.core)
+            (let  [result   (do-search (ExecutionEngine. database) query)]
```
Comment (jexp) [inline language-guides/clojure/server-extension/src/example/core.clj]: don't re-create execution-engines, but have them injected from jersey as `@Context CypherExecutor`

File: language-guides/clojure/server-extension/src/example/core.clj [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                  cc/generate-string
+                  (Response/ok)
+                  .build)))
+         (^{GET true
+            Produces ["text/plain"]
+            Path "/graph"}
+          graph
+          [this ^{Context true} database ^{QueryParam "limit"} limit]
+            (require 'example.core)
+            (let  [result  (get-graph (ExecutionEngine. database) limit)]
```
Comment (jexp) [inline language-guides/clojure/server-extension/src/example/core.clj]: see above

File: language-guides/clojure/server-extension/src/example/core.clj [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                  cc/generate-string
+                  (Response/ok)
+                  .build)))
+         (^{GET true
+            Produces ["text/plain"]
+            Path "/{title}"}
+          findMovie
+          [this ^{Context true} database ^{PathParam "title"} query]
+            (require 'example.core)
+            (let  [result   (find-movie (ExecutionEngine. database) query)]
```
Comment (jexp) [inline language-guides/clojure/server-extension/src/example/core.clj]: see above

File: language-guides/clojure/server-extension/src/example/core.clj (hunk 1 above)
Comment (ducky427) [inline language-guides/clojure/server-extension/src/example/core.clj]: I've fixed that in the new p-r.

**Comment by kbastani** [comment]
Please provide an update on this PR. Is it ready to be merged in?

**Comment by jexp** [comment]
@ducky427 perhaps you could extend the readme a bit, talking about what it takes to make this work well with clojure? in terms of JAX-RS / Jersey integration.

**Comment by ducky427** [comment]
@kbastani  @jexp I will add more documentation to the readme. I'll also add links to other clojure<->java JAX-RS interop resources.

The tricky thing has been to get leiningen to create the uberjar with all the required necessary dependencies. It took me a good 6 months to get the process right as this is a very narrow field of an already narrow language.

**Comment by whatSocks** [comment]
@ducky427 any update on this?
:)

**Pull request closed without merging**